provisioning/miniprov/mypy.ini

```diff
@@ -0,0 +1,8 @@
+[mypy]
+python_version = 3.6
```

Comment:
Please look into https://github.com/Seagate/cortx-hare/blob/main/hax/mypy.ini

There is an important directive there:
```
check_untyped_defs = True
```
that appeared when it turned out that mypy (by default) doesn't try to validate the source code which IS NOT type-hinted. We missed some issues because of that. Please see [this doc](https://mypy.readthedocs.io/en/stable/common_issues.html#no-errors-reported-for-obviously-wrong-code) for more details.

Reply:
Done